neo4j-store/src/main/kotlin/org/ostelco/prime/storage/graph/Neo4jStore.kt

```diff
@@ -1520,7 +1529,7 @@ object Neo4jStoreSingleton : GraphStore {
                     }
                     PaymentStatus.REQUIRES_PAYMENT_METHOD -> {
                         NotCreatedError(type = planEntity.name, id = "Failed to subscribe $customerId to ${plan.id}",
```

Comment:
NotCreatedError should have `type = "Customer subscription to Plan"` and `id = "$customerId -> ${plan.id}"`.

Reply:
Fixed.